prosessering-core/src/main/kotlin/no/nav/familie/prosessering/util/TaskPrioritet.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.prosessering.util
+
+import org.slf4j.MDC
+
+internal object TaskPrioritet {
+    const val MDC_TASK_PRIORITET = "taskPrio"
```

Comment:
Trengs dette? Er det så ofte vi vil avvike fra normalen at vi ikke bare bør sette det eksplisitt? Føler at dette kan bli en bortgjemt funksjonalitet som er lett å misse for den gjengse utvikler 😱

Reply:
Det er kun task-prosessering som må forholde seg til denne. Ikke ef-sak eller ba-sak
Dette la jeg til fordi jeg tenker at det er fint at vi "sender videre prioriteten" på task-flytet fra en oppgave til en annen, for å ikke trenge å sette det rundt om i koden, se eksemplet i beskrivelsen til PR, og tester :)